### ISSUE TITLE
Update JEDI-Eval rocoto to handle Orion memory changes

### ISSUE THREAD

**Issue by CoryMartin-NOAA** [NOAA-EMC/JEDI-Eval#49]
From Vincent Sanders:
```
On Wednesday, October 6, beginning at 1:00pm CDT, changes to orion's slurm configuration will be made.
These changes will be completed live and a short window of service interruption for job submissions may occur while the scheduler is restarted.

These changes should help with job throughput.

Users should note a change to the default memory allocation.
The default memory allocation per core will be changed from using all available memory to being based on cores requested per node.
Standard compute nodes w/ 192GB will default to 4608 Mb per core
Big mem nodes w/ 384GB will default to 9472 Mb per core
Users can change these defaults by using the --mem or --exclusive options.

Running jobs should not be affected, and queued jobs may have their priority/fairshare adjusted after the
reconfiguration.
```

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA I think this is our first thing to iron out. When I try to use `iasi_metop-a` along with additional observation types, the job fails. Running with just `iasi_metop-a` it completes. Going from 2 nodes, 12 tasks per node to 4, 6, still results in the same error. 

I think we need to do something additional for rocoto to make sure there is enough memory available. I tried adding `<memory>31G</memory>` with no/limited success.

**Comment by RussTreadon-NOAA** [comment]
Thanks @CoryMartin-NOAA for the heads up.   I'll look into this.

**Comment by RussTreadon-NOAA** [comment]
Changing 
`        <!ENTITY RESOURCES_HOFX_GDAS "<nodes>2:ppn=12:tpp=1</nodes>">`
to 
`        <!ENTITY RESOURCES_HOFX_GDAS "<nodes>12:ppn=2:tpp=1</nodes>">`

allows gdas_hofx to run to completion with 
```
obs_list:
- $<< $(bundle)/ufo/ewok/jedi-gdas/amsua_n19.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/aircraft.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/ompstc8_npp.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/iasi_metop-a.yaml
```
The question now becomes how to specify resource settings in a general, yet job specific, manner.

**Comment by RussTreadon-NOAA** [comment]
```
        <!ENTITY RESOURCES_HOFX_GDAS "<nodes>2:ppn=12:tpp=1</nodes>">
        <!ENTITY NATIVE_HOFX_GDAS    "--mem=0">
```
works.   As explained in the slurm documentation:
> NOTE: A memory size specification of zero is treated as a special case and grants the job access to all of the memory on each node for newly submitted jobs and all available job memory to new job steps.

**Comment by CoryMartin-NOAA** [comment]
@RussTreadon-NOAA excellent, I think `<!ENTITY NATIVE_HOFX_GDAS    "--mem=0">` is what I was looking for but wasn't sure what the exact option was.

**Comment by RussTreadon-NOAA** [comment]
Create branch [feature/hofxmem](https://github.com/NOAA-EMC/JEDI-Eval/tree/feature/hofxmem) as copy of JEDI-Eval:develop at 998385d.    Replace `--export=NONE` for `NATIVE_HOFX_GDAS` in `hofx.xml` template with `--mem=0`.   Tested this change on Orion for 2020121506 through 2020121600 with the following hofx configuration
```
        <!ENTITY WALLTIME_HOFX_GDAS  "01:00:00">
        <!ENTITY RESOURCES_HOFX_GDAS "<nodes>2:ppn=12:tpp=1</nodes>">
        <!ENTITY NATIVE_HOFX_GDAS    "--mem=0">
```
and 
```
obs_list:
- $<< $(bundle)/ufo/ewok/jedi-gdas/amsua_n19.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/aircraft.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/ompstc8_npp.yaml
- $<< $(bundle)/ufo/ewok/jedi-gdas/iasi_metop-a.yaml
```
`gdas_hofx` ran to completion.

**Comment by CoryMartin-NOAA** [comment]
Closed through PR #65
